VideoLocker/src/main/java/org/edx/mobile/util/FileUtil.java

```diff
@@ -38,4 +41,25 @@
             inputStream.close();
         }
     }
+
+    /**
+     * Deletes a single file and/or recursively deletes files/folders within a folder before
+     * deleting it.
+     *
+     * @param fileOrDirectory The file/folder that needs to be deleted.
+     * @param exceptions      Name of the file/folder that needs to be skipped while deletion.
```

Comment:
You shouldn't be sloppy when writing utility code, and this change isn't too hard to implement. But the final decision is up to you.